### ISSUE TITLE
Idea for: Multi-Language Learn English App

### ISSUE THREAD

**Issue by tcburns24** [codeforsanjose/Project-Ideas#97]
Hi all, I'm looking to recruit a few hackers to get this idea off the ground. Here's the info:

**1) The Big Idea**
Anyone can use Google translate to look up words, but what if there was an app that offered quick, concentrated language practice in an entertaining, gameified way? That's what this app is aiming for — not a Google Translate replacement, but a tool for English Language Learners can use where the work of determining vocab has already been done. 

**2) What's the Vocab?**
To determine vocab, I want to start by thinking of everyday actions. For example, a trip to the grocery store requires on-sight recognition of "Self Checkout", "Baked Goods", "Dairy". Getting a refill at the pharmacy requires knowledge of "Drop-off", "Pick-up", "Signature Required", etc.. This is how I'm planning to generate the vocabulary. How many words in total? I haven't decided yet, probably 50-75 to start, then we can think about expanding after that. 

**3) Gameified**
Users are introduced to a set of related vocab words (8-12 words at a time) on a practice page. They can interact with the vocab to embed it into their memory. From there, the user will be redirected to a test page where he attempts to recall the translation. The user's success rate will be tracked, so he can attempt to beat prior scores. 

**In a nutshell**
I want to build something that empowers adult English Language Learners. Without a strong English proficiency, simple everyday tasks can be intimidating and cumbersome. I feel like this app has the power to strip the anxiety and difficulty out of such tasks, and ultimately make the Bay Area (or anywhere) a more welcoming place for non-native English speakers. 

If you're interested in teaming up with me on this project, email me at **tcburns24@gmail.com**.
I'm hoping to recruit some hackers with backend proficiency. Thanks!
